### ISSUE TITLE
Problem with the new front end

### ISSUE THREAD

**Issue by Nekodificador** [coolzilj/ComfyUI-LJNodes#13]
I just found that LJNodes mess with the new search bar. When you double clic on the UI for use the search, it open both at the same time.

![image](https://github.com/user-attachments/assets/2f88821f-2f85-4aa1-ba68-21597760cdf1)

**Comment by oliverban** [comment]
Yes, I was just about to come here and say the same. Would love for this to be remedied, love this pack otherwise! :)

**Comment by makeitrad** [comment]
I can confirm this as well. Just uninstalled and things work now.

**Comment by zhouyc98** [comment]
I made a fork and fixed it, and only kept the "Add Reroute in between" functionality.  
You can try it here: https://github.com/zhouyc98/ComfyUI-LJNodes
